rdkit/DataStructs/VectCollection.py

```diff
@@ -169,13 +168,19 @@ def NumChildren(self):
   def GetChildren(self):
     return tuple(self.__vects.items())
 
-  def GetBit(self, id):
+  def __getitem__(self, idx):
     if self.__needReset:
       self.Reset()
-    return self[id]
+    return self.__orVect.GetBit(idx)
```

Comment:
I moved the definition of __getitem__ up and use 
``` GetBit = __getitem__ ```
instead of defining a function.